### PR TITLE
Remove user-select

### PR DESCRIPTION
**Pull request by amanmahajan7**

### CODE REVIEW

File: src/style/core.ts [hunk 1]
```diff
@@ -51,7 +51,6 @@ const root = css`
   border: 1px solid var(--rdg-border-color);
   box-sizing: border-box;
   overflow: auto;
-  user-select: none;
```
Comment (amanmahajan7) [inline src/style/core.ts:54]: Double clicking on editable cell selects the text first and then the cell goes into edit mode and the editor value is selected again. Not the bet UI but I haven't found a good solution

**Comment by nstepien** [comment]
Should we maybe disable text selection on editable cells?

**Comment by amanmahajan7** [comment]
I thought about it but the formatter may display a different value for example if we have complex editor like a modal.

**Comment by amanmahajan7** [comment]
I think we can merge this and revisit if there are any issues

**Review by nstepien — APPROVED** [review]

**Pull request merged**